src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/HttpHandlerDiagnosticListener.cs

```diff
@@ -0,0 +1,672 @@
+                }
+            }
+            public override bool IsFixedSize
+            {
+                get
+                {
+                    return this._table.IsFixedSize;
+                }
+            }
+            public override bool IsSynchronized
+            {
+                get
+                {
+                    return this._table.IsSynchronized;
+                }
+            }
+            public override object this[object key]
+            {
+                get
+                {
+                    return this._table[key];
+                }
+                set
+                {
+                    this._table[key] = value;
+                }
+            }
+            public override object SyncRoot
+            {
+                get
+                {
+                    return this._table.SyncRoot;
+                }
+            }
+            public override ICollection Keys
+            {
+                get
+                {
+                    return this._table.Keys;
+                }
+            }
+            public override ICollection Values
+            {
+                get
+                {
+                    return this._table.Values;
+                }
+            }
+            internal HashtableWrapper(Hashtable table) : base()
+            {
+                this._table = table;
+            }
+            public override void Add(object key, object value)
+            {
+                this._table.Add(key, value);
+            }
+            public override void Clear()
+            {
+                this._table.Clear();
+            }
+            public override bool Contains(object key)
+            {
+                return this._table.Contains(key);
+            }
+            public override bool ContainsKey(object key)
+            {
+                return this._table.ContainsKey(key);
+            }
+            public override bool ContainsValue(object key)
+            {
+                return this._table.ContainsValue(key);
+            }
+            public override void CopyTo(Array array, int arrayIndex)
+            {
+                this._table.CopyTo(array, arrayIndex);
+            }
+            public override object Clone()
+            {
+                return new HashtableWrapper((Hashtable)this._table.Clone());
+            }
+            IEnumerator IEnumerable.GetEnumerator()
+            {
+                return this._table.GetEnumerator();
+            }
+            public override IDictionaryEnumerator GetEnumerator()
+            {
+                return this._table.GetEnumerator();
+            }
+            public override void Remove(object key)
+            {
+                this._table.Remove(key);
+            }
+            public override void OnDeserialization(object sender)
+            {
+            }
+
+            private const string ParentTableSerializationField = "ParentTable";
```

Comment:
Argh, I missed this. Good catch.
Fixed now, and removed both this const and the OnDeserialization method.